x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -65,7 +65,16 @@ export const securityRuleTypeFieldMap = {
 
 /* eslint-disable complexity */
 export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
-  ({ lists, logger, config, ruleDataClient, ruleExecutionLoggerFactory, version, isPreview }) =>
+  ({
+    lists,
+    logger,
+    config,
+    publicBaseUrl,
```

Comment:
Does the use of `publicBaseUrl` require customers to configure the `core.http.basePath.publicBaseUrl` setting in order for alert URLs to work?

Rules created through the UI have a 
```
meta: {from: "1m", kibana_siem_app_url: "http://localhost:5601/jde/app/security"}
``` 
object where the `kibana_siem_app_url` is populated from the browser to provide `results_link` functionality. Should we use that param in conjunction with the publicBaseUrl?

Reply:
Note from discussing on zoom: `kibana_siem_app_url` does not get populated for prebuilt rules at the moment. We may want to update the prebuilt rule installation to populate that field for prebuilt rules.

Reply:
> We may want to update the prebuilt rule installation to populate that field for prebuilt rules

I'm missing the context @marshallmain -- could you please explain what this field is used for?

Reply:
It's added to the actions context to provide users of 3rd party systems with a link back to the alerts table. However, since the Kibana server does not always know the public facing URL of Kibana we rely on this `kibana_siem_app_url` rule parameter that is set by the UI when custom rules are created.